django_statsd/models.py

```diff
@@ -38,3 +39,21 @@ def model_delete(sender, **kwargs):
 if getattr(settings, 'STATSD_MODEL_SIGNALS', False):
     post_save.connect(model_save)
     post_delete.connect(model_delete)
+
+
+def logged_in(sender, request, user, **kwargs):
+    statsd.incr('auth.login.success')
+    statsd.incr('auth.backends.%s' % user.backend.replace('.', '_'))
+
+
+def logged_out(sender, request, user, **kwargs):
+    statsd.incr('auth.logout.success')
+
+
+def login_failed(sender, credentials, **kwargs):
+    statsd.incr('auth.login.failed')
+
+if getattr(settings, 'STATSD_AUTH_SIGNALS', False):
```

Comment:
I'm going to say it's fine though it should be in the `apps.py`

Reply:
Sure, maybe put it into an issue to move it later on.